internal/gengapic/genrest.go

```diff
@@ -529,24 +529,85 @@ func (g *generator) pagingRESTCall(servName string, m *descriptor.MethodDescript
 	p := g.printf
 
 	inType := g.descInfo.Type[m.GetInputType()].(*descriptor.DescriptorProto)
-	// outType := g.descInfo.Type[m.GetOutputType()].(*descriptor.DescriptorProto)
+	outType := g.descInfo.Type[m.GetOutputType()].(*descriptor.DescriptorProto)
 
 	inSpec, err := g.descInfo.ImportSpec(inType)
 	if err != nil {
 		return err
 	}
 
-	// outSpec, err := g.descInfo.ImportSpec(outType)
-	// if err != nil {
-	// 	return err
-	// }
+	outSpec, err := g.descInfo.ImportSpec(outType)
+	if err != nil {
+		return err
+	}
+	info, err := getHTTPInfo(m)
+	if err != nil {
+		return err
+	}
+	if info == nil {
+		return errors.E(nil, "method has no http info: %s", m.GetName())
+	}
+	max := "math.MaxInt32"
+	ps := "int32(pageSize)"
```

Comment:
Why add these to variables? It doesn't save much, using literal values in the print statements is fine?

Reply:
Copy/paste mode :P